commitizen/commands/bump.py

```diff
@@ -228,15 +232,27 @@ def __call__(self):  # noqa: C901
 
             # Increment is removed when current and next version
             # are expected to be prereleases.
-            if prerelease and current_version.is_prerelease:
-                increment = None
+            force_bump = False
+            if current_version.is_prerelease:
+                last_final = self.find_previous_final_version(current_version)
+                if last_final is not None:
+                    commits = git.get_commits(last_final)
+                    increment = self.find_increment(commits)
+                    semver = last_final.increment_base(
+                        increment=increment, force_bump=True
+                    )
+                    if semver != current_version.base_version:
```

Comment:
To keep the same naming convention (and try to contain semver wording to the version scheme implementation)

```suggestion
                    base = last_final.increment_base(
                        increment=increment, force_bump=True
                    )
                    if base != current_version.base_version:
```

Reply:
Commit https://github.com/commitizen-tools/commitizen/pull/799/commits/9da9302445fe0ac5a4169dd0759c243a6885b28c.